service.rb

```diff
@@ -27,7 +27,7 @@ def initialize(port)
         end
         response_message << "</ul>\n"
         response_message << daily_data_form
-      when ["POST", "/add-daily-data"]
+      when ["POST", "/show/data"]
```

Comment:
Because if I had it as `/add-daily-data`, I couldn't redirect the page back to `/show/data` and thought this was the right way to do it (lol). I kinda thought of it like Rail models method's new/create/destroy action methods, where `/show/data` is the data model, and POST and GET is the action. 

What is the alternate way?

Reply:
Okay, I understand. In general you can make whatever decision you want to here because the computer doesn’t care. Given that this is an educational exercise I think there‘s some value in having different paths for the sake of clarity — it’s more obvious that the path `/add-daily-data` is doing something other than showing the data — but it’s not important really.

That being said, my [comment below](https://github.com/wildmaples/mirth/pull/1#discussion_r555759218) gives some alternatives, and hints at a practical reason why you might want this to live at a different path and provide some other way for the user to get back to the original path.